packages/portal-proto/src/components/SaveSetButton.tsx

```diff
@@ -75,7 +77,7 @@ const SaveSetButton: React.FC<SaveSetButttonProps> = ({
         onClose={() => setShowSaveModal(false)}
         onActionClick={(name: string) => {
           setSetName(name);
-          createSet({ values: ids });
+          createSet({ values: ids, intent: "user", set_type: "mutable" });
```

Comment:
<img width="1138" alt="Screenshot 2024-04-05 at 2 54 15 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/cd064d1b-6c82-48b2-a00a-07b17d68096d">

Reply:
ephemeral - we don't guarantee these sets hang out forever